calculation/sha_calc/sha_calc/directivity/bea20/scripts/std_mean_plots.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import common
+
+
+def std_mean_plots(
+    baseline_dir,
+    input_dir,
+    faults,
+    nhyps,
+    nhm_dict,
+    grid_space,
+    colour_mode,
+    output_dir,
+):
+    for fault_name in faults:
+
+        fault, _, planes, lon_lat_depth, x, y = common.load_fault_info(
+            fault_name, nhm_dict, grid_space
+        )
+
+        modes = ["std", "mean bias"]
+
+        baseline_array = np.load(f"{baseline_dir}/{fault_name}_20000_fd.npy").reshape(
+            (10000, 1)
+        )
+
+        for mode in modes:
+            fig = plt.figure(figsize=(16, 10))
+
+            for i, nhyp in enumerate(nhyps):
+                fd_array = np.load(f"{input_dir}/{fault_name}_{nhyp}_fd_mc.npy")
+                ratio = np.log(fd_array / baseline_array)
+
+                if mode == "std":
+                    values = np.std(ratio, axis=0).reshape((grid_space, grid_space))
+                else:
+                    values = np.mean(ratio, axis=0).reshape((grid_space, grid_space))
+
+                ax1 = fig.add_subplot(2, 2, i + 1)
+                if mode == "std":
+                    bounds = list(np.round(np.linspace(0, 0.05, 13), 3))
+                    if colour_mode == "contour":
+                        c = plt.contourf(
+                            x,
+                            y,
+                            values,
+                            levels=bounds,
+                            cmap=plt.cm.get_cmap("Reds", 12),
+                            extend="max",
+                        )
+                    elif colour_mode == "mesh":
+                        c = ax1.pcolormesh(
+                            x, y, values, cmap="Reds", vmax=0.05, vmin=0, shading="flat"
+                        )
+                else:
+                    bounds = list(np.round(np.linspace(-0.01, 0.01, 21), 3))
+                    colour_map = [
+                        "#ff0000",
+                        "#ff1b1b",
+                        "#ff3636",
+                        "#ff5151",
+                        "#ff6b6b",
+                        "#ff8686",
+                        "#ffa1a1",
+                        "#ffbcbc",
+                        "#ffd7d7",
+                        "#ffffff",
+                        "#ffffff",
+                        "#d7d7ff",
+                        "#bcbcff",
+                        "#a1a1ff",
+                        "#8686ff",
+                        "#6b6bff",
+                        "#5151ff",
+                        "#3636ff",
+                        "#1b1bff",
+                        "#0000ff",
+                    ]
+                    colour_map.reverse()
```

Comment:
Why not have it reversed to start with? (or at least reverse in the same line)

Reply:
For some reason having reverse() on the list caused issues, not sure why, but yea have just put in the reverse order to start with now.